docs/source/integrations/databricks_deployment_workflow.md

```diff
@@ -0,0 +1,316 @@
+```
+
+### Authenticate the Databricks CLI
+
+**Now, you must authenticate the Databricks CLI with your Databricks instance.**
+
+[Refer to the Databricks documentation](https://docs.databricks.com/dev-tools/cli/index.html#set-up-authentication) for a complete guide on how to authenticate your CLI. The key steps are:
+
+1. Create a personal access token for your user on your Databricks instance.
+2. Run `databricks configure --token`.
+3. Enter your token and Databricks host when prompted.
+4. Run `databricks fs ls dbfs:/` at the command line to verify your authentication.
+
+### Create a new Kedro project
+
+Create a Kedro project with the PySpark Iris starter using the following command in your local environment:
+
+```bash
+kedro new --starter=pyspark-iris
+```
+
+Name your new project `iris-databricks` for consistency with the rest of this guide. This command creates a new Kedro project using the PySpark Iris starter template.
+
+### Create an entry point for Databricks
+
+The default entry point of a Kedro project uses a Click command line interface (CLI), which is not compatible with Databricks. To run your project as a Databricks job, you must define a new entry point speficially for use on Databricks.
+
+The PySpark Iris starter has this entry point pre-built, so there is no extra work to do in this guide. **Generally you must create it manually for your own projects using the following steps**:
+
+1. **Create an entry point script**: Create a new file in `<project_root>/src/iris_databricks` named `databricks_run.py`. Copy the following code to this file:
+
+```python
+import argparse
+import logging
+
+from kedro.framework.project import configure_project
+from kedro.framework.session import KedroSession
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--env", dest="env", type=str)
+    parser.add_argument("--conf-source", dest="conf_source", type=str)
+    parser.add_argument("--package-name", dest="package_name", type=str)
+
+    args = parser.parse_args()
+    env = args.env
+    conf_source = args.conf_source
+    package_name = args.package_name
+
+    # https://kb.databricks.com/notebooks/cmd-c-on-object-id-p0.html
+    logging.getLogger("py4j.java_gateway").setLevel(logging.ERROR)
+    logging.getLogger("py4j.py4j.clientserver").setLevel(logging.ERROR)
+
+    configure_project(package_name)
+    with KedroSession.create(env=env, conf_source=conf_source) as session:
+        session.run()
+
+
+if __name__ == "__main__":
+    main()
+```
+
+2. **Define a new entry point**: Open `<project_root>/src/setup.py` in a text editor or IDE and add a new line in the definition of the `entry_point` tuple, so that it becomes:
+
+```python
+entry_point = (..., "databricks_run = <package_name>.databricks_run:main")
+```
+
+Remember to replace <package_name> with the correct package name for your project.
+
+This process adds an entry point to your project which can be used to run it on Databricks.
+
+```{note}
```

Comment:
I need to check if the bullet formatting you've added here in the note will work when built by Sphinx...